.travis.yml

```diff
@@ -97,6 +97,7 @@ before_install:
       echo extension = redis.so >> $INI
       echo extension = memcached.so >> $INI
       echo extension = mongodb.so >> $INI
+      echo extension = gmp.so >> $INI
```

Comment:
there is no gmp.so on Travis. So this is breaking the CI.

gmp is already available on Travis, as it is built as a bundled extension

Reply:
i'll remove it too